app/models/invitation.rb

```diff
@@ -73,7 +73,10 @@ def send_and_set_date
           # send invitations actively sent by a user synchronously to avoid delays
           UserMailer.invitation(self.id).deliver_now
         end
-        self.sent_at = Time.now
+
+        # We cannot simple use "self.sent_at = Time.now" since that change is not persisted when this
```

Comment:
I'd prefer to have a comment like "Skip callbacks within after_save to avoid a callback loop" instead, because my first thought on seeing `update_column` is ["why not `update`?"](https://stackoverflow.com/questions/22196318/rails-after-save-infinite-loop).

Reply:
I've merged your version of the comment and some of the SO post replies you linked.
I hope this makes the most sense to future readers!